### PR TITLE
Build category and rules page

### PR DESCRIPTION
**Pull request by ishitakeshawani**
- Added quiz by  category
- Added rules page
Preview link: https://deploy-preview-2--memory-nomads.netlify.app/

### CODE REVIEW

File: src/pages/categories-page/CategoryPage.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import { React } from "react";
```
Comment (Naveenchandar) [inline src/pages/categories-page/CategoryPage.js]: React is default export and not a named export from react. Please update it.

File: src/pages/categories-page/CategoryPage.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  };
+
+  return (
+    <div>
+      <div className="quiz-category flex-row">
+        <div className="quiz-category-title">{categoryName} Quizzes</div>
+        <div className="flex-row quiz-category-list">
+          {getQuizList().map((quiz) => {
+            return (
+              <div className="card card-box-shadow">
```
Comment (Naveenchandar) [inline src/pages/categories-page/CategoryPage.js]: key prop is missing. While mapping the data, please add key prop to the parent element.

**Review by Naveenchandar — APPROVED** [review]

File: src/pages/homepage/homepage.css [hunk 3]
```diff
@@ -1,5 +1,5 @@
 .quiz-home-page {
-  justify-content: space-around;
+  justify-content: space-around !important;
```
Comment (SanketDhabarde) [inline src/pages/homepage/homepage.css]: Try to solve with specificity, instead of an important tag

**Review by SanketDhabarde — APPROVED** [review]
- Good work with UI🔥
- Like that, you wrote your own controller

File: src/pages/categories-page/CategoryPage.js (hunk 1 above)
Comment (ishitakeshawani) [reply src/pages/categories-page/CategoryPage.js]: done

File: src/pages/categories-page/CategoryPage.js (hunk 2 above)
Comment (ishitakeshawani) [reply src/pages/categories-page/CategoryPage.js]: done

File: src/pages/homepage/homepage.css (hunk 3 above)
Comment (ishitakeshawani) [reply src/pages/homepage/homepage.css]: done

**Pull request merged**